### PR TITLE
VIH-7312 Update otherinformation format handling

### PR DESCRIPTION
**Pull request by virensareen**
### JIRA link (if applicable) ###
VIH-7312

### Change description ###
Changed otherinformation format handing from JSON string to vertical pipe-separated string

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs [hunk 1]
```diff
@@ -50,19 +50,43 @@ public static bool DoesJudgePhoneExist(this HearingDetailsResponse hearing)
 
         public static string GetJudgeContactEmail(this HearingDetailsResponse hearing)
         {
-            return GetOtherInformationObject(hearing.Other_information).JudgeEmail;
+            var email = GetOtherInformationObject(hearing.Other_information).JudgeEmail;
+            if (email == "")
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs:54]: use `string.empty`

File: AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs [hunk 2]
```diff
@@ -50,19 +50,43 @@ public static bool DoesJudgePhoneExist(this HearingDetailsResponse hearing)
...
+                return null;
+            }
+            return email;
         }
         
         public static string GetJudgePhone(this HearingDetailsResponse hearing)
         {
-            return GetOtherInformationObject(hearing.Other_information).JudgePhone;
+            var phone = GetOtherInformationObject(hearing.Other_information).JudgePhone;
+            if (phone == "")
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs:64]: `string.empty`

**Review by RuwanikaWeerakkody — DISMISSED** [review]

**Review by shaed-parkar — COMMENTED** [review]
Looks good, just needs a bit of tidying up. Did you check notify?

**Review by shaed-parkar — APPROVED** [review]
LGTM

**Review by RuwanikaWeerakkody — APPROVED** [review]

**Pull request merged**